force-app/main/default/lwc/patientLookup/patientLookup.js

```diff
@@ -33,19 +33,60 @@ export default class PatientLookup extends LightningElement {
     }
 
     get patientRecordTypeId() {
-        return Object.values(this.accountObjInfo.data.recordTypeInfos).find(rti => rti.name=='Patient').recordTypeId;
+        return Object.values(this.contactObjInfo.data.recordTypeInfos).find(rti => rti.name=='Master').recordTypeId;
     }
 
     handleFormChange(event) {
         this.patient[event.currentTarget.dataset.field] = event.target.value;
-        this.publishChange(this.patient);
+
+        this.template.querySelector('.btn-lookup').disabled
+            = !this.patient.Patient_Identifier__pc;
     }
 
-    handleLookup() {
+    async handleLookup() {
+        this.template.querySelector('.btn-lookup').disabled = true;
+
+        this.patient = {
+            RecordTypeId: this.patientRecordTypeId,
+            Patient_Identifier__pc: this.patient.Patient_Identifier__pc
+        };
+
+        this.patientProvider = await findPatient({
+            phn: this.patient.Patient_Identifier__pc,
+        });
+
+        if (this.patientProvider.gender == undefined,
+            this.patientProvider.dob == undefined,
+            this.patientProvider.familyName == undefined,
+            this.patientProvider.givenName == undefined)
+        {
+            // No Data!
+            this.hasData = false;
+            this.completeAndNoResults = true;
+        } else {
+            this.completeAndNoResults = false;
+            this.hasData = true;
 
+            this.patient = {
+                FirstName: this.patientProvider.givenName,
+                LastName: this.patientProvider.familyName,
+                Gender: this.patientProvider.gender,
+                Deceased: this.patientProvider.deceased,
+                PersonBirthdate: new Date(this.patientProvider.dob),
+                ...this.patient
+            }
```

Comment:
@marklise Could you set a `verified` flag to `this.patient` as well? 

For reference, I was also setting a `verified` flag [on the provider as well based on their `status`](https://github.com/bcgov/MoH-SAT/blob/8a0ce9c1a9e91f300268b46a0da308dee48c686e/force-app/main/default/lwc/providerLookup/providerLookup.js#L81). The flag will be used in the Create SA Request flow, including the [validation on the odrLookup component](https://github.com/bcgov/MoH-SAT/blob/8a0ce9c1a9e91f300268b46a0da308dee48c686e/force-app/main/default/lwc/odrLookup/odrLookup.js#L37).